content/faq.md

```diff
@@ -1,123 +1,202 @@
+{{% /faq-box %}}
 
-Yes! We have 100GBASE-CWDM4 we can give you free of charge, and we are happy to
-provide other transceivers at cost price.
+---
 
-### Why do you have sites that are invite only?
+## Technical Questions
 
-It boils down to what connectivity we can manage to secure from our sponsors.
-If a site is known to have large networks, we need to be selective on which
-networks we on-board.
+### ➔ Where are your locations?
 
-Want to help? Have a lookt at our [donation](/donate/) page to see what kind
-of equipment we need to make the site better.
+{{% faq-box %}}
+SONIX operates in these locations across Sweden:
 
-### Can I use your metrics for my own monitoring?
+| **City**     | **Locations**                                                                                           |
+|--------------|---------------------------------------------------------------------------------------------------------|
+| **Stockholm** | [Obe Kista Gate](https://peeringdb.com/fac/5544), [Stokab KN7](https://peeringdb.com/fac/1053), [Digital Realty Stockholm](https://peeringdb.com/fac/208), [Equinix SK1](https://peeringdb.com/fac/156), [City Hosting](https://peeringdb.com/fac/14462) |
+| **Gothenburg** | [Gothnet DC Marieholm](https://peeringdb.com/fac/434), [Gothnet DC Backa](https://peeringdb.com/fac/9308) |
+| **Malmö**      | [Västergatan 4](https://peeringdb.com/fac/484) |
+{{% /faq-box %}}
 
-Absolutely! We expose a Prometheus-compatible API at `https://metric.sonix.network/api/v1/`.
+### ➔ Do you have Route Servers?
 
-Let us know if you end up using it, we would love to learn if people find it useful.
+{{% faq-box %}}
+Yes, we encourage all peers to connect:
+
+| **Location**         | **ASN**  | **IPv4**          | **IPv6**                  |
+|-----------------------|----------|-------------------|---------------------------|
+| SONIX Stockholm #1    | AS61229  | `185.1.215.1`     | `2001:7f8:117::6:1229:1`  |
+| SONIX Malmö #1        | AS61229  | `193.201.148.129` | `2001:7f8:117:2:0:6:1229:1` |
+
+We are working on deploying additional route servers and redundancy.
+{{% /faq-box %}}
+
+### ➔ What best practices do you recommend?
+
+{{% faq-box %}}
+- Follow our [rules](/rules/).
+- Configure BGP timers: **3s Keepalive** and **9s Hold**.
+{{% /faq-box %}}
+
+### ➔ What MTU is used?
+
+{{% faq-box %}}
+We use **MTU 9000**.
+{{% /faq-box %}}
+
+### ➔ Do you allow Transit Peering over the IX?
+
+{{% faq-box %}}
+Yes, settlement-free peering is allowed, including free IPv6 transit.
+{{% /faq-box %}}
+
+### ➔ What media do you support?
+
+{{% faq-box %}}
+We support the following transceivers:
+
+| **Speed** | **Transceivers**                       |
+|-----------|----------------------------------------|
+| 400G      | 400GBASE-LR4, 400GBASE-FR4            |
+| 200G      | 200GBASE-LR4, 200GBASE-FR4            |
+| 100G      | 100GBASE-CWDM4, 100GBASE-LR4, LR1     |
+| 50G       | 50GBASE-LR, 50GBASE-ER                |
```

Comment:
How about we remove the 50G option and reuse the 50GBASE-ER transceivers internally? @bluecmd

Reply:
No matter what, it's only the 100G transcievers that are mentioned to be in stock, see below:
`We recommend 100GBASE-CWDM4 (available for free).`

So I think it's fine to showcase that it's possible, even though unlikely. Thus, leaving usage free of choice internally or to customers based on availability, right?